lib/timeutils/unixtime.c

```diff
@@ -368,3 +368,22 @@ unix_time_to_unix_epoch(const UnixTime ut)
 {
   return (guint64)((ut.ut_sec + ut.ut_gmtoff) * USEC_PER_SEC + ut.ut_usec);
 }
+
+UnixTime
+unix_time_add_unix_time(UnixTime time, UnixTime add)
+{
+  UnixTime ut =
+  {
+    .ut_sec = time.ut_sec + add.ut_sec,
+    .ut_usec = time.ut_usec + add.ut_usec,
+    .ut_gmtoff = time.ut_gmtoff, //TODO: figure this out
```

Comment:
Or maybe we should also fail in case both datetime's have timezones. If we do that, essentially this becomes a datetime + timedelta (or timedelta + datetime) operation, as the only difference between a datetime and timedelta is that timedelta cannot have a timezone.

Reply:
this is the case when we use rhs datetime as timedelta, simply skipping its timezone.

Reply:
We decided not to support adding datetimes. We will eventually have a duration type that could be added. Also we will have a `combine_date_and_time()` function or sth similar for combining two datetimes.